src/govuk/components/header/header.mjs

```diff
@@ -30,12 +48,23 @@ Header.prototype.init = function () {
  *
  * Sync the menu button class and the accessible state of the menu and the menu
  * button with the visible state of the menu
- *
- * @param {boolean} isVisible Whether the menu is currently visible
  */
-Header.prototype.syncState = function (isVisible) {
-  this.$menuButton.classList.toggle('govuk-header__menu-button--open', isVisible)
-  this.$menuButton.setAttribute('aria-expanded', isVisible)
+Header.prototype.syncState = function () {
```

Comment:
Think the comment for this function could do with updating.

Reply:
Updated. Let me know if it covers what the function is doing.